packages/edit-site/src/components/layout/style.scss

```diff
@@ -259,25 +259,3 @@ html.canvas-mode-edit-transition::view-transition-group(toggle) {
 	padding-left: 16px;
 	padding-right: 16px;
 }
-
-.edit-site-layout.is-full-canvas {
-	.edit-site-layout__view-mode-toggle.components-button {
```

Comment:
So far as I can tell, `.edit-site-layout__view-mode-toggle` will never render inside `.edit-site-layout.is-full-canvas`.